swan-lake/by-example/graphql-file-upload/content.jsx

```diff
@@ -42,10 +42,10 @@ service /fileUpload on new graphql:Listener(4000) {
         return "Successfully Uploaded";
     }
 
-    // Remote functions in GraphQL services can use the \`graphql:Upload[]\` as an input parameter
-    // type. Therefore, remote functions can accept an array of \`graphql:Upload\` values. This can
+    // remote methods in GraphQL services can use the \`graphql:Upload[]\` as an input parameter
```

Comment:
```suggestion
    // Remote methods in GraphQL services can use the \`graphql:Upload[]\` as an input parameter
```

Reply:
Need to change in the BBE source in the dist repo during the BBE hackathon.